mods/d2k/rules/defaults.yaml

```diff
@@ -267,6 +272,7 @@
 	ShakeOnDeath:
 	Sellable:
 		SellSounds: BUILD1.WAV
+	AutoTargetIgnore:
```

Comment:
Strange.  AFAIK we haven't changed anything relevant to AutoTarget or the AI since the failure from #3337 / #3467.